src/app/models/LikeModel.ts

```diff
@@ -0,0 +1,23 @@
+import * as mongodb from 'mongodb';
+
+class LikeModel {
+ _id: mongodb.ObjectId;
+ email: string;
+ planId: mongodb.ObjectId; 
+ createdAt: Date;
+
+ constructor(
+    email = '',
+    planId = '', 
+    createdAt = new Date(),
+
```

Comment:
extra line not needed and can be deleted

Reply:
removed in latest commit to branch